app/public/components/buttons/copyLinkButton.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import snackBar from '../snackBars/snackBar.js';
+
+/**
+ * @param {string} url - url that will be returned to the user
```

Comment:
I think that can also be URL object, thus `${<string>url}` === `${<URL>url}`

Reply:
No, it is always a string here.